src/proxy/ClientTable.php

```diff
@@ -107,65 +107,108 @@ public function getOffsetTotal()
     
     public function isComplet()
     {
-        return $this->getRows() == count($this->getContentRows());
+        return $this->getRows() == $this->_contentRowsCount;
     }
     
-    private $_contentRows;
-    
     public function getContentRows()
```

Comment:
It should not be a problem to drop this method, no need for backwards compatibility support.